openshift/e2e-common.sh

```diff
@@ -252,18 +252,19 @@ function create_configmaps(){
   # when KO_DATA_PATH was overwritten.
   oc create configmap ko-data-eventing -n $OPERATORS_NAMESPACE --from-file="openshift/release/knative-eventing-ci.yaml" || return $?
 
+  # TODO: Remove this sed line when serverless-operator starts using operator 0.24. KOURIER_GATEWAY_NAMESPACE should be set by operator.
+  sed -i -e 's/value: \"kourier-system\"/value: \"knative-serving-ingress\"/g' third_party/kourier-latest/kourier.yaml || return $?
```

Comment:
I just wanted make it clear. We need this manual `sed` until serverless-operator includes this code  https://github.com/knative/operator/blob/13ae9e144766c70b05d484a4c915d8463cff0ed9/pkg/reconciler/knativeserving/ingress/kourier.go#L37
Current downstream serverless-operator only reconciles `3scale-kourier-control` deployment.